packages/retail-ui-extensions/src/extension-api/navigation-api/navigation-api.ts

```diff
@@ -7,6 +7,9 @@ export interface NavigationApiContent {
   /** The parameters the screen was displayed with */
   params?: any;
 
+  /** Pops the currently presented screen */
```

Comment:
Should we use a more meaningful word than "Pops" for 3P developers? Most of these people likely are not mobile devs

Reply:
I think `pop` works as it is an underlaying functionality for any browser navigation.

Reply:
@samiulmonir06 I didn't know that, then pop it must be 😄

Reply:
For the record I wasn't saying we should rename the function, just the language in the comment